### ISSUE TITLE
overdot rendering in word doc

### ISSUE THREAD

**Issue by pressler-vsc** [metanorma/metanorma-ieee#418]
When I use the `metanorma/metanorma:1.10.8` docker image to compile equations with overdots, the equations render incorrectly in the word document:

![image](https://github.com/user-attachments/assets/e9c6055a-0471-4469-94c3-6847593e7fc3)

It appears the dot and symbol positions are transposed?

The rendering in the PDF appears as expected:

![image](https://github.com/user-attachments/assets/bceff4d6-5fd5-4157-b1fe-9953c9a48857)

I produced the above by appending the following source to `https://github.com/metanorma/mn-samples-ieee/blob/main/sources/p987.6/sections/04-important_elements.adoc`:

```asciidoc
[latexmath]
++++
\dot{\Theta} \neq \ddot{\Theta}
++++

[%key]
latexmath:[\dot{\Theta}]:: is theta dot
latexmath:[\ddot{\Theta}]:: is theta double-dot

[asciimath]
++++
dot Theta ne ddot Theta
++++

[%key]
asciimath:[dot Theta]:: is theta dot
asciimath:[ddot Theta]:: is theta double-dot
```

and rendering the document via

```shell
mkdir -p build
docker run --rm -v "$(pwd)":/metanorma/ -w /metanorma metanorma/metanorma \
    metanorma compile \
              -t=ieee \
              -o=build \
              --agree-to-terms \
              sources/p987.6/document.adoc 
```

**Comment by opoudjis** [comment]
Word uses a distinct markup language for maths than PDF and HTML (MathML vs OOXML), so that is plausible. Will confirm and pass on to my colleague Suleman to fix if so, as this is managed in a separate piece of code.

**Comment by suleman-uzair** [comment]
@opoudjis Just fixed this in PR plurimath/plurimath#300, see the screenshot and the generated OMML below.

![Screenshot 2024-10-03 at 12 33 23 PM](https://github.com/user-attachments/assets/828d93a6-2bef-4ad3-beeb-21344b3a2f45)
